### PR TITLE
Correct alignment of right arrow on number scrubber

### PR DESCRIPTION
**Pull request by kevinb7**
Addresses issue https://github.com/Khan/live-editor/issues/241.

### CODE REVIEW

File: css/ui/tooltips.css [hunk 1]
```diff
@@ -399,8 +399,11 @@ html.playing .tooltip.hide-while-playing {
 
 .scrubber-handle .flipped-arrow {
     display: inline-block;
-    transform: rotate(180deg);
-    -webkit-transform: rotate(180deg);
+    transform: scaleX(-1.0);
+    -webkit-transform: scaleX(-1.0);
+    -moz-transform: scaleX(-1.0);
+    -ms-transform: scaleX(-1.0);
+    -o-transform: scaleX(-1.0);
```
Comment (kevinb7) [inline css/ui/tooltips.css]: I'd like to suggest migrating to stylus + nib for doing CSS.  nib provides auto-prefixing and stylus could simplify our nested rules and provide nice things like variables for common colors and stuff.

http://learnboost.github.io/stylus/
http://nibstyl.us/docs/#transparent-mixins

Comment (pamelafox) [inline css/ui/tooltips.css]: We use LESS internally at KA, so that's what we'd want to move to if we migrated this repo over.

Comment (kevinb7) [inline css/ui/tooltips.css]: I found this auto-prefixer for LESS: http://lessprefixer.com/.  Is this what other repos are using for auto-prefixing?

Comment (kevinb7) [inline css/ui/tooltips.css]: I also found https://github.com/postcss/autoprefixer which doesn't require either LESS (or stylus).  It also appears to be much better maintained than lessprefixer.

Comment (pamelafox) [inline css/ui/tooltips.css]: Hm. It's possible we don't actually have an autoprefixing system in place internally. I'm poking around.

Comment (pamelafox) [inline css/ui/tooltips.css]: Aha, we use https://github.com/postcss/autoprefixer, it gets auto-run over our code.

**Pull request merged**